.github/workflows/release.yml

```diff
@@ -54,6 +56,20 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+      - name: Build system-logger arm64
```

Comment:
There's no need to add new part here. Just add it to the previous "platform:" part with comma in between.

Reply:
Did that it gives error " ERROR: docker exporter does not currently support exporting manifest lists
Error: buildx failed with: ERROR: docker exporter does not currently support exporting manifest lists"

Reply:
Tried that it gives this error:
``ERROR: docker exporter does not currently support exporting manifest lists``